js/envato-market.js

```diff
@@ -236,6 +238,44 @@
       regex = new RegExp( '[\\?&]' + name + '=([^&#]*)' );
       results = regex.exec( location.search );
       return null === results ? '' : decodeURIComponent( results[1].replace( /\+/g, ' ' ) );
+    },
+
+    maybeLoadHelthcheck: function() {
+      // We only load the health check ajax call when the envato-market-healthcheck div is visible on the page.
+      var $healthcheckOutput = $( '.envato-market-healthcheck' );
+      if( $healthcheckOutput.is( ':visible') ) {
+        $healthcheckOutput.text('Loading...');
+
+        // Use our existing wp.ajax.post pattern from above to call the healthcheck API endpoint
+        var request = wp.ajax.post( _envatoMarket.action + '_healthcheck', {
+          nonce: _envatoMarket.nonce
+        });
+
+        request.done(function( response ) {
+          if( response && response.limits ) {
+            var $healthcheckUL = $( '<ul></ul>' );
+            var limits = Object.keys( response.limits );
+            for( var i = 0; i < limits.length; i++ ) {
+              var $healcheckLI = $( '<li></li>' );
```

Comment:
Typo: should this be `healthcheckLI`?

Reply:
Yes again :+1: fixed: https://github.com/envato/wp-envato-market/pull/214/commits/989eacc937515bf41e5789bd1fe10ae444cb1788